### ISSUE TITLE
Maintainers Wanted

### ISSUE THREAD

**Issue by gsferreira** [TalksLab/metro-bootstrap#65]
Unfortunately, the former maintainers no longer have the time and/or resources to work on this project. This means that bugs will not be fixed and features will not be added unless someone else does so. 

If you're interested in fixing up metro-bootstrap, please reply to this issue.

**Comment by Jiasm** [comment]
I hope to be able to participate in this awesome project

**Comment by fskreuz** [comment]
I'm interested :D

**Comment by gsferreira** [comment]
Good news! We have been discussing with @fskreuz by email and now he is part of the Team.
Welcome @fskreuz!

**Comment by codefactorymx** [comment]
I would like to help

**Comment by gsferreira** [comment]
@codefactorymx could you contact me on _me@gsferreira.com_?

**Comment by PsiACE** [comment]
Wanted to be part of the project.

**Comment by gsferreira** [comment]
Cool! @PsiACE  could you contact me on me@gsferreira.com?

**Comment by PsiACE** [comment]
@gsferreira Of course, my e-mail address is psiace@outlook.com. You probably got two identical e-mails. I'm sorry I sent them twice.

**Comment by prathamesh-borse** [comment]
I'm Interested Can I contribute to this Project as a maintainer?

**Comment by gsferreira** [comment]
@prathamesh-borse Thanks for that! Added to the Team! 🎉

**Comment by garritfra** [comment]
Hi!

I just saw your entry on the [Maintainers Wanted](https://github.com/pickhardt/maintainers-wanted) list. I just published a very similar project that's now looking for postings. If this project is still looking for maintainers, I'd greatly appreciate if you find some time to check out this project, and maybe add yours to the list.

Here's the link to the list:
https://seeking-maintainers.net

Thanks for your time!

**Comment by Bashamega** [comment]
Dear Mr. Ferreira,

My name is Adam, and I am a front-end developer with a passion for open source and GitHub. I would be honored to take on the role of maintaining the Metro Bootstrap project. I believe that with consistent maintenance and development, this project has significant potential for growth.

I have experience in maintaining open-source projects, and you can check out my work on my [GitHub](https://github.com/bashamega) and [personal website](https://adambashaahmednaji.com/). I would be excited to contribute to the continued success of Metro Bootstrap.

Thank you for considering my request. I look forward to your response.

Sincerely,
Adam

**Comment by gsferreira** [comment]
@Bashamega  Thank you! Added to the Team! 🎉

**Comment by Bashamega** [comment]
@gsferreira I think the maintainer note should be removed, since i and the team will revive it. Also is there a better and easier way for communicating with you? May be discord or slack

**Comment by gsferreira** [comment]
> @gsferreira I think the maintainer note should be removed, since i and the team will revive it. Also is there a better and easier way for communicating with you? May be discord or slack

what's your discord username?

**Comment by Bashamega** [comment]
> > @gsferreira I think the maintainer note should be removed, since i and the team will revive it. Also is there a better and easier way for communicating with you? May be discord or slack
> 
> what's your discord username?

Mine is bashamega. With the same pfp

**Comment by Bashamega** [comment]
Also I have my discord server on my readme. You can join it if you want

**Comment by Bashamega** [comment]
Also I have my discord server on my readme. You can join it if you want

**Comment by Anujsharma002** [comment]
I am interested

**Comment by Bashamega** [comment]
> I am interested
Feel free to start contributing

**Comment by Anujsharma002** [comment]
@Bashamega  Thanks! I'll take a look at the open issues and start with something small first to get familiar with the codebase. Is there a "good first issue" you'd recommend, or should I just pick from the 3 open ones? Also happy to join the Discord if that's easier for coordinating.

**Comment by Bashamega** [comment]
> @Bashamega  Thanks! I'll take a look at the open issues and start with something small first to get familiar with the codebase. Is there a "good first issue" you'd recommend, or should I just pick from the 3 open ones? Also happy to join the Discord if that's easier for coordinating.

There is no discord, also there is not a lot of issues. You can check the code and see if there is improvement s

**Comment by Anujsharma002** [comment]
@Bashamega  ok sounds good
